src/gpl/README.md

```diff
@@ -34,17 +34,24 @@ weight nets with low slack. It adjusts the worst slacks (modified with
 `-timing_driven_net_weight_max`). The multiplier
 is scaled from the full value for the worst slack, to 1.0 at the
 `timing_driven_nets_percentage` point. Use the `set_wire_rc` command to set
-resistance and capacitance of estimated wires used for timing.
+resistance and capacitance of estimated wires used for timing. 
+
+Timing-driven iterations are triggered based on a list of overflow threshold 
+values. Each time the placer execution reaches these overflow values, the 
+resizer is executed. This process can be costly in terms of runtime. The 
+default overflow values for recalculating weights are [79, 64, 49, 29, 21, 15] 
+and can be modified with `-timing_driven_net_reweight_overflow`, you may use 
+less overflow threhsold values to decrease runtime, for example.
 
 When the routability-driven option is enabled, each of its iterations will 
-execute FastRoute to provide an estimation of routing congestion, which is 
-typically costly in terms of runtime. Congested tiles will have the area of 
-their logic cells inflated to reduce routing congestion. The iterations will 
-attempt to achieve the target RC (routing congestion) by comparing it to the 
-final RC. If the algorithm takes too long during routability-driven execution, 
-consider raising the target RC value to alleviate the constraints. The final 
-RC value is calculated based on the weight coefficients. The algorithm will 
-stop if the RC is not decreasing for three consecutive iterations.
+execute RUDY to provide an estimation of routing congestion. Congested tiles 
+will have the area of their logic cells inflated to reduce routing congestion. 
+The iterations will attempt to achieve the target RC (routing congestion) 
+by comparing it to the final RC at each iteration. If the algorithm takes too 
+long during routability-driven execution, consider raising the target RC value 
```

Comment:
as above. Reduce guesswork; be explicit about which option to use to "the target RC value".